adadamp/_dist.py

```diff
@@ -447,3 +452,28 @@ def score(self, X, y=None):
         self._meta.update(stat)
         self._meta["score__calls"] += 1
         return acc
+
+class DaskClassifierIncreasingLR(DaskClassifier):
+    
+    def fit(self, X, y=None, **fit_params):
+        
+        if not hasattr(self, "initialized_") or not self.initialized_:
+            self.initialize()
+            
+        self.curr_metas = []
+        
+        print('Initial Learning Rate:', self.optimizer_.param_groups[0]['lr'])
+        
+        for epoch in range(1, self.max_epochs + 1):
+            
+            if epoch % 60 == 0:
+                for g in self.optimizer_.param_groups:
+                    g['lr'] = g['lr'] / 5
+                    print('Updated Learning Rate:', g['lr'])
+                
+            self.partial_fit(X, y=y, **fit_params)
+            acc = self.score(X, y=y)
+            
+            self.curr_metas.append(deepcopy(self._meta))
```

Comment:
Does the stats need to record number of workers? It is not currently

Reply:
Good idea!